scripts/globals/mobskills.lua

```diff
@@ -514,7 +514,7 @@ end
 -- Equation: (HP * percent) + (LVL / base)
 -- cap is optional, defines a maximum damage
 xi.mobskills.mobBreathMove = function(mob, target, percent, base, element, cap)
-    local damage = (mob:getHP() * percent) + (mob:getMainLvl() / base)
+    local damage = mob:getHP() * percent
```

Comment:
this should be based off of wyvern's max HP, and not current HP.
https://ffxiclopedia.fandom.com/wiki/Healing_Breath#:~:text=Amount%20of%20HP%20cured%20is%20based%20on%20the%20wyvern%27s%20maximum%20HP.%20Also%2C%20while%20TP%20is%20not%20required%20for%20use%2C%20each%20Healing%20Breath%20will%20fully%20consume%20the%20wyvern%27s%20TP.%20Every%2020%25%20of%20wyvern%27s%20TP%20will%20slightly%20raise%20the%20amount%20healed.%0AThe%20current%20working%20formula%20for%20Healing%20Breath%20III%20is%3A

Reply:
This is not the wyvern code this is for mobskills only:
https://w.atwiki.jp/studiogobli/pages/22.html